packages/client/src/pages/Forum/index.ts

```diff
@@ -0,0 +1 @@
+export { Forum as default } from './Forum'
```

Comment:
Можно так сделать. Можешь по желанию исправить, но необязательно
```suggestion
export { default } from './Forum'
```

Reply:
export { default } from './Forum' если я не ошибаюсь, но при условии что сам компонент Forum будет экспортирован тоже по дефолту. Типа 
const Forum =() => {
  return <h1>Forum</h1>
}
export default Forum 
Ну это заглушки, походу подправим

Reply:
А, да, не обратил внимание, что там именованный экспорт.